builtin/aws/ecs/platform.go

```diff
@@ -3215,6 +3221,9 @@ deploy {
 
 	doc.SetField("health_check",
 		"Health check settings for the app.",
+		docs.Summary("These settings configure a health check for the application"+
```

Comment:
```suggestion
		docs.Summary("These settings configure a health check for the application "+
```

Reply:
I think these words will get smooshed together without this space.

Reply:
Fixed! 😄